### ISSUE TITLE
Stats

### ISSUE THREAD

**Issue by smockeh** [CCLA-Will-Check/ccla-wills#58]
Is there a quick way we can get how many wills are in the database, and how many requests have been made? Bonus points if we could specify time frame, but overall numbers are good too!

**Comment by ghost** [comment]
@smockeh I've added a display for the number of wills on the wills index page. I'll look into putting something like that in for the number of requests. For time frame, do you mean e.g. "wills added between this date and this date" or "requests made between this date and this date" or both?

**Comment by smockeh** [comment]
Yeah, both! Would be for reporting to the Board purposes. Not a huge priority though.
